install/index.md

```diff
@@ -33,7 +38,7 @@ Once the CSM installation has completed, other product streams for the HPE Cray
 
 The topics in this chapter need to be done as part of an ordered procedure so are shown here with numbered topics.
 
-**Note**: If problems are encountered during the installation, some topics have their own troubleshooting
+**Note**: If problems are encountered during the installation, some topics have their own [troubleshooting sections found in the operations index](../operations/index.md)
```

Comment:
- This line is missing a period
- Could add backticks to **`Note`** to be consistent with the other changes

Reply:
Corrected